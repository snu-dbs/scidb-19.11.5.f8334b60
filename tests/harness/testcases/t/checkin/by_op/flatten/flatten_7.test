--setup
--start-query-logging
store(build(<v:int64>[i=0:2,1,0,j=0:2,2,0],3*i+j), REPL, distribution: replicated)

--test
show(REPL)
op_sum(flatten(REPL),v)

--cleanup
remove(REPL)
--stop-query-logging
