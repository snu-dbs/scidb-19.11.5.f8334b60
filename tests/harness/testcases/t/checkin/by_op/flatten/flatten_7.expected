SCIDB QUERY : <store(build(<v:int64>[i=0:2,1,0,j=0:2,2,0],3*i+j), REPL, distribution: replicated)>
Query was executed successfully

SCIDB QUERY : <show(REPL)>
{i} schema,distribution,etcomp
{0} 'REPL<v:int64> [i=0:2:0:1; j=0:2:0:2]','replicated','none'

SCIDB QUERY : <op_sum(flatten(REPL),v)>
{i} v_sum
{0} 36

SCIDB QUERY : <remove(REPL)>
Query was executed successfully

